app/jobs/runtime/delete_expired_package_blob.rb

```diff
@@ -36,6 +32,18 @@ def max_attempts
         def logger
           @logger ||= Steno.logger('cc.background')
         end
+
+        private
+
+        def package_registry_configured?
+          VCAP::CloudController::Config.config.package_image_registry_configured?
+        end
+
+        def create_package_source_deletion_job(package)
```

Comment:
Why is this same method defined in two different places?

Reply:
They're private methods for each class.

I introduced them to take the focus off of the messy conditionals in the `perform` method. My hope was that it would make it clearer what was going on in `perform`. 

However, I made them private because I don't want them to become part of the public interface of these jobs and want them to remain free to drift over time.

At one point while extracting this logic out of the main `perform` methods these actually had different code in them. They only became the same when I got fancy and wanted to use the `&.` operator. 🙂